sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -26,6 +35,12 @@ namespace pi {
     SYCL_BE_PI_OTHER
   };
 
+#ifdef SYCL_RT_OS_WINDOWS
+#define PLUGIN_NAME "pi_opencl"
```

Comment:
let's have it be "libpi_opencl.dll"

>>>If the string specifies a module name without a path and the file name extension is omitted, the function appends the default library extension .dll to the module name. To prevent the function from appending .dll to the module name, include a trailing point character (.) in the module name string

So add the "." in the implementation of loadOsLibrary for Windows

Reply:
i'll change it to pi_opencl.dll?
The windows platform generates the .dll with the same name as pi_opencl. It does not append lib to pi_opencl